### DISCUSSION TITLE
Issues with Install - Linux Lite

### DISCUSSION THREAD

**Discussion by JoshsGitRDone** [pymedusa/Medusa#11814] [Q&A]
Hello,

Not sure if I put this in the right place but I’m having issues with getting Medusa to work.

I have the most current version of Linux Lite installed and if I understand it’s based on Ubuntu.

In terminal I’ve checked and I’m supposed to use systemd instructions.

I’ve also installed node.js and followed all directions found here https://github.com/pymedusa/Medusa/wiki/Medusa-installation-Debian-Ubuntu?new_signup=true

Any help would be greatly appreciated bc every time
I check status for Medusa it says failed even though I followed all directions and copied and pasted so know I didn’t mistype anything.

thanks

**Comment by p0psicles** [comment]
Try to start medusa manually. Using `python start.py` or `python3 start.py`. With the same user/group you have in your systemd config. And see where it goes wrong.
